### PR TITLE
Release 0.18.0

### PR DESCRIPTION
**Pull request by zimlu02**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 # Changelog
 All notable changes to the COBOL Language Support extension are documented in this file.
 
+# [0.18.0](https://github.com/eclipse/che-che4z-lsp-for-cobol/compare/0.17.1...0.18.0) (2021-03-12)
+
+### Bug Fixes
+* Add color to hover message ([f15e867](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f15e867c0d3af03f75012b40ca6c83360e04f35e))
+* Add coloring for divisions ([988fd45](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/988fd457fe447b7c84846301e423648485314fca)), closes [#796](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/796)
+* Fix misspelled Keyword Warning Shown on a Correct Token [#760](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/760) ([8958921](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/8958921e2073847ab1cccb344556e491cf502c34))
```
Comment (temanbrcom) [inline CHANGELOG.md:9]: Something strange is going on with case here

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+### Features
+* Add COBOL world pattern to IDE editor ([37213a7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/37213a7ae6dd26b8367e3f5cc4a26d9bf3392928)), closes [#798](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/798)
```
Comment (temanbrcom) [inline CHANGELOG.md:12]: Fix autocomplete for identifiers separated with dashes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+* Add IDMS DML support for DB statements part 1 [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([#790](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/790)) ([08574f7](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/08574f7b53728e20b49cf2c9aedb10d7e75d0b5d))
```
Comment (temanbrcom) [inline CHANGELOG.md:13]: Without "part 1"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,21 @@
...
+* Add support for IDMS DML commands [#704](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/704) ([f8ca612](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/f8ca61261fe9a737bcdd7289c094040e0aa5b6bb))
+* Hover over variable shows its hierarchy definition ([d7084ec](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/d7084ec5a227f09e8a6e99592ce2af71569088be)), closes [#757](https://github.com/eclipse/che-che4z-lsp-for-cobol/issues/757)
```
Comment (temanbrcom) [inline CHANGELOG.md:15]: Make hovering over a variable show its hierarchy definition

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,21 @@
...
+* SKIP compiler directive can be terminated with a separator period. ([ef52f7e](https://github.com/eclipse/che-che4z-lsp-for-cobol/commit/ef52f7e07ae5a7bdf8cdd9e93c9eb2782649f991))
```
Comment (temanbrcom) [inline CHANGELOG.md:16]: Fix terminating SKIP compiler directive with a period

File: CHANGELOG.md (hunk 2 above)
Comment (temanbrcom) [inline CHANGELOG.md:12]: Should we leave "closes" here?

Comment (zeibura) [reply CHANGELOG.md:12]: What is this a comment on? Is it an extra PR which didnt make it into the autogenerated list?

File: CHANGELOG.md (hunk 5 above)
Comment (zeibura) [reply CHANGELOG.md:16]: Also moving this to bug fixes

File: CHANGELOG.md (hunk 2 above)
Comment (zeibura) [reply CHANGELOG.md:14]: Remove it if you want

Comment (temanbrcom) [reply CHANGELOG.md:14]: It is a description of this feature from the user point of view

Comment (zeibura) [reply CHANGELOG.md:14]: OK done

**Review by temanbrcom — APPROVED** [review]
🍰

**Pull request merged**